typescript/web/src/pages/[workspaceSlug]/datasets/index.tsx

```diff
@@ -55,6 +62,7 @@ const Body = () => {
 
   return (
     <>
+      {shouldDisplayUpdateBanner && <UpdatePlanBanner />}
```

Comment:
As it'll possibly be reused in many pages, the boolean logic should be placed within UpdatePlanBanner which will take care of the responsibility to add the necessary components or not.

Reply:
Possibly.
But for now, there is an advantage to use this way: you don't pollute virtual DOM with useless components
<img width="1678" alt="Capture d’écran 2022-02-24 à 17 54 31" src="https://user-images.githubusercontent.com/43350109/155570718-27a1842a-1db1-4eb4-a245-de99b9110212.png">

Reply:
It will not pollute the DOM. I've just asked for something like:

```tsx
const UpdatePlanBanner = () => {
  const showBanner = // get the info
  return <>{showBanner && <UpdatePlanBannerComponent /></>
}
```

There is no extra component added with such code. The `<>` fragment does not creates any DOM component,

Reply:
I've already updated it